compose/install/compose-plugin.md

```diff
@@ -0,0 +1,194 @@
+For other Linux installation methods see [this summary](#installation-methods).
+
+>To run Compose as a non-root user, see [Manage Docker as a non-root user](../../engine/install/linux-postinstall.md){:target="_blank" rel="noopener" class="_"}.
+
+
+If you have already set up the Docker repository jump to step 2.
+
+1. Set up the repository. Go to the "Set up the repository" section of the chosen [Linux distribution](../../engine/install/index.md#server){:target="_blank" rel="noopener" class="_"}. found in the Docker Engine installation pages to check the instructions.
+
+2. Update the `apt` package index, and install the _latest version_ of Docker Compose:
+
+    > Or, if using a different distro, use the equivalent package manager instructions. 
+
+
+    ```console
+    $ sudo apt-get update
+    $ sudo apt-get install docker-compose-plugin
+    ```
+    
+    Alternatively, to install a specific version of Compose CLI plugin:
+      
+    a. List the versions available in your repo:
+
+
+      ```console
+      $ apt-cache madison docker-compose-plugin
+
+        docker-compose-plugin | 2.3.3~ubuntu-focal | https://download.docker.com/linux/ubuntu focal/stable arm64 Packages
+      ```
+
+    b. From the list obtained use the version string you can in the second column to specify the version you wish to install. 
+      
+    c. Install the selected version:
+
+
+      ```console
+      $ sudo apt-get install docker-compose-plugin=<VERSION_STRING>
+      ```
+    where `<VERSION_STRING>` is, for example,`2.3.3~ubuntu-focal`.
+
+3.  Verify that Docker Compose is installed correctly by checking the version.
+
+    ```console
+    $ docker compose version
+    Docker Compose version v2.3.3
+    ```
+
+### Install the plugin manually
+
+> **Note**
+>
+> These instructions assume you already have Docker Engine and Docker CLI installed and now want to install the Compose plugin. 
+>
+> Note as well this option requires you to manage upgrades manually. Whenever possible we recommend any of the other installation methods listed. For other Linux installation methods see [this summary](#installation-methods).
+
+>To run Compose as a non-root user, see [Manage Docker as a non-root user](../../engine/install/linux-postinstall.md).
+
+
+1.  To download and install the Compose CLI plugin, run:
+
+    ```console
+    $ DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
+    $ mkdir -p $DOCKER_CONFIG/cli-plugins
+    $ curl -SL https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-linux-x86_64 -o $DOCKER_CONFIG/cli-plugins/docker-compose
+    ```
+
+    This command downloads the latest release of Docker Compose (from the Compose releases repository) and installs Compose for _the active user_ under `$HOME` directory. 
+    
+    > To install:
+    >* Docker Compose for _all users_ on your system, replace `~/.docker/ cli-plugins` with `/usr/local/lib/docker/cli-plugins`.
```

Comment:
Found a minor typo in the first directory, removing one space.

```suggestion
    >* Docker Compose for _all users_ on your system, replace `~/.docker/cli-plugins` with `/usr/local/lib/docker/cli-plugins`.
```

Reply:
Thanks, for your comments Stefan. I have already negotiated each of Usha's suggestions offline. I have been using the "Resolve conversation" button bellow each comment to mark changes I have already addressed in my local branch before committing. So all comments are still present on the page but just collapsed.
I will add your improvements as well and then address conflicts.

Reply:
I also can't see this change. Please take another look.